docker_images/sklearn/app/pipelines/text_classification.py

```diff
@@ -0,0 +1,113 @@
+import json
```

Comment:
I think we should refactor so that this file doesn't repeat as much code as it is right now from the `tabular-classification` task.